### PR TITLE
Locksmith endpoint for saving events

### PR DESCRIPTION
**Pull request by benwerd**
# Description

Adds endpoint to re-save an event once it has been created.

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [x] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [x] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: locksmith/src/operations/eventOperations.ts [hunk 1]
```diff
@@ -14,6 +16,22 @@ namespace EventOperations {
     return Event.create(event)
   }
 
+  export const update = async (event: EventCreation): Promise<any> => {
+    event.owner = Normalizer.ethereumAddress(event.owner)
+    event.lockAddress = Normalizer.ethereumAddress(event.lockAddress)
+    return await Event.update(event, {
+      where: {
+        lockAddress: {
+          [Op.eq]: ethJsUtil.toChecksumAddress(event.lockAddress),
```
Comment (akeem) [inline locksmith/src/operations/eventOperations.ts]: Lets us` Normalizer.ethereumAddress` it should functionally the be the same as `ethJsUtil.toChecksumAddress` but should help us to localize the conversion

File: locksmith/src/operations/eventOperations.ts [hunk 2]
```diff
@@ -14,6 +16,22 @@ namespace EventOperations {
...
+        },
+        owner: {
+          [Op.eq]: ethJsUtil.toChecksumAddress(event.owner),
```
Comment (akeem) [inline locksmith/src/operations/eventOperations.ts]: same as the other address above

File: locksmith/__tests__/controllers/eventController.test.ts [hunk 3]
```diff
@@ -108,6 +108,38 @@ describe('Event Controller', () => {
       })
     })
   })
+  describe('event update', () => {
+    describe('when unaccompanied by a valid signature', () => {
```
Comment (akeem) [inline locksmith/__tests__/controllers/eventController.test.ts]: How would you feel about adding a test case here for the scenario where the signature is valid but the person is attempting to update a lock that they may not own?

Comment (benwerd) [reply locksmith/__tests__/controllers/eventController.test.ts]: I'd feel good about it! Test added.

**Pull request merged**